### PR TITLE
token-client: Add fmt::Display implementation to RpcClientResponse

### PR DESCRIPTION
**Pull request by joncinque**
#### Problem

While working on #3593, I wanted to print the result from the token client, but the Debug printer for `Signature` gives `Signature(signature_in_base_58)`, which is not great.

#### Solution

Implement `fmt::Display` on `RpcClientResponse`, and add a feature for output that can be disable if needed.

### CODE REVIEW

File: update-solana-dependencies.sh [hunk 1]
```diff
@@ -47,4 +47,5 @@ crates=(
 set -x
 for crate in "${crates[@]}"; do
   sed -E -i'' -e "s:(${crate} = \")(=?)${old_solana_ver}\".*:\1\2${solana_ver}\":" "${tomls[@]}"
+  sed -E -i'' -e "s:(${crate} = \{ version = \")(=?)${old_solana_ver}(\".*):\1\2${solana_ver}\3:" "${tomls[@]}"
```
Comment (2501babe) [inline update-solana-dependencies.sh:50]: hahaha, i wasnt fast enough to sneakily commit my driveby, i wanted to test it first
```
-  sed -E -i'' -e "s:(${crate} = \")(=?)${old_solana_ver}\".*:\1\2${solana_ver}\":" "${tomls[@]}"
+  sed -E -i'' -e "s:(${crate} = )(\{ version = ){0,1}(\")(=?)${old_solana_ver}\".*:\1\2\3\4${solana_ver}\":" "${tomls[@]}"
```

Comment (joncinque) [reply update-solana-dependencies.sh:50]: Hehe, thanks! looks like we pretty much got to the same place, right?

**Review by 2501babe — APPROVED** [review]

File: update-solana-dependencies.sh (hunk 1 above)
Comment (2501babe) [reply update-solana-dependencies.sh:50]: ya its equivalent
